src/Cake.Npm/BumpVersion/NpmBumpVersionSettingsExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Cake.Npm.BumpVersion
+{
+    using System;
+
+    /// <summary>
+    /// Extensions for <see cref="NpmBumpVersionSettings"/>.
+    /// </summary>
+    public static class NpmBumpVersionSettingsExtensions
+    {
+        /// <summary>
+        /// Defines that npm version should commit, even if repository is not clean
+        /// exists on disk.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="force">Whether to set force, or not.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmBumpVersionSettings.Force"/> set.</returns>
+        public static NpmBumpVersionSettings WithForce(this NpmBumpVersionSettings settings, bool force = true)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            settings.Force = force;
+            return settings;
+        }
+
+        /// <summary>
+        /// Sets the commit message 
+        /// exists on disk.
```

Comment:
Something seems wrong with the second part of this sentence

Reply:
✔️